packages/modules/storybook-auth-addon/src/index.js

```diff
@@ -1,3 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 import { BrowserAuthorizationCallbackHandler } from "@bentley/frontend-authorization-client";
 
-BrowserAuthorizationCallbackHandler.handleSigninCallback(window.location.toString()).catch(console.log);
+// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Is this from a different change? What does nullish coalescing has to do here ?

Reply:
Ah should be my eslint extension. Removed